i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md

```diff
@@ -43,7 +43,7 @@ Momento Cache はこの古典的なサーバーレスの定義に完璧に適合
 
 第二に、Momento Cache はキャッシュ管理に関する設定を抽象化しています。キャッシュに必要なインスタンスの数を指定することも、最大メモリサイズを考えることも必要ありません。Momento はクラウドに特化して作られていて、現代的なクラウド基盤の弾力性やスケーラビリティを最大活用しています。皆さんはキャッシュに必要なデータをどれだけでも保存することができて、Momento がそれをシームレスに取り扱います。
 
-最後に、Momento Cache は従量課金のモデルです。上述の様に、利用の有無に関係なく、事前に特定のキャッシュの種類やクラスターサイズを選択することはありません。Momento は Momento Cache と Momento Topics で行われたデータ転送の出入りに対して[課金します](../manage/pricing)。それ以外は全て含まれています。これによって、料金は皆さんの支配下にあって、アプリケーションの変更が直接的に請求金額に影響を与えます。
+最後に、Momento Cache は従量課金のモデルです。上述の様に、利用の有無に関係なく、事前に特定のキャッシュの種類やクラスターサイズを選択することはありません。Momento は Momento Cache と Momento Topics で行われたデータ転送の出入りに対して[課金します](https://www.gomomento.com/pricing)。それ以外は全て含まれています。これによって、料金は皆さんの支配下にあって、アプリケーションの変更が直接的に請求金額に影響を与えます。
```

Comment:
FYI the English pricing page is as above but the path is different on the Japanese page (`pricing` vs `price`):

https://www.gomomento.com/pricing

vs

https://jp.gomomento.com/price/

Reply:
Good catch, will fix.